demo/app/views/auto_complete_test/index.html.erb

```diff
@@ -13,3 +14,8 @@
     <% end %>
   <% end %>
 <% end %>
+<% else %>
+  <%= render(Primer::Beta::AutoComplete::Item.new(role: "none", "aria-hidden": true, value: "", disabled: true)) do %>
+      No results!
+  <% end %>
```

Comment:
What would you think about making another class for the "no results" case that passes in these arguments? Eg.

```ruby
module Primer
  module Alpha
    class AutoComplete < Primer::Component
      class NoResultsItem < Item
        def initialize(**system_argments)
          super(
            role: :none,
            aria: merge_aria(
              { aria: { hidden: true } },
              system_arguments
            ),
            value: "",
            disabled: true,
            **system_arguments
          )
        end
      end
    end
  end
end
```

Users would then render like this:

```erb
<%= render(Primer::Beta::AutoComplete::NoResultsItem.new) do %>
  No results!
<% end %>
```

Reply:
🤔 LOVE IT!